site/content/docs/user/private-registries.md

```diff
@@ -90,6 +91,24 @@ rm $HOME/.docker/config.json
 [-f $HOME/.docker/config.json.host] && mv $HOME/.docker/config.json.host $HOME/.docker/config.json
 ```
 
+### ... with Service Accounts
+
+Access tokens are short lived though, so you may prefer to use a Service Account and keyfile.
+First, either download the key from the console or generate one with gcloud:
+
+```
+gcloud iam service-accounts keys create <output.json> --iam-account <account email>
+```
+
+Then, replace the `gcloud auth...` line from the previous snippet with:
```

Comment:
`from the previous section`
?

Reply:
Yea, I probably should be more direct and anchor the section

> Then, replace `gcloud auth print-access-token | ...` from the [previous section] (#use-an-access-token) with:

Or maybe you have another way to format the reference without copying the entire snippet?